src/commands/gate/README.md

```diff
@@ -1 +1,62 @@
-WORK IN PROGRESS
\ No newline at end of file
+# GATE command
+
+Evaluate Quality Gates rules in Datadog.
```

Comment:
I feel like the description is too short.
Do you think we can add a short description of what exactly it's doing?
It might be useful to add that it returns different exit codes and can be used to gate (pass/fail) a pipeline based on the data in Datadog.

Also, maybe it's worth to add that it should be called in the pipeline after sending data to Datadog.

Reply:
Just for context, this won't be the primary source for understanding how to setup quality gates, but we will have a dedicated documentation page, similarly to the `junit` command: https://docs.datadoghq.com/continuous_integration/tests/junit_upload/?tab=linux. 

The description here is short and aligned with Sarif command ("Upload your SARIF report files.") and junit command ("Upload your jUnit XML files."). I don't think we need much more than this as we'll have documentation, that for example will cover the fact that the events needs to be sent before the command is executed, etc.

Reply:
I agree we don't need to go into detail for this README, but the exit code bit might be small and useful. 

"The exit code of this command will change depending on the result of the evaluation rule"? or something like that

Reply:
👍 Added